### ISSUE TITLE
Default hardfork

### ISSUE THREAD

**Issue by ryanio** [ethereumjs/ethereumjs-vm#634]
The README currently states:

```
Fork Support
The VM currently supports the following hardfork rules:

Byzantium
Constantinople
Petersburg (default)
Istanbul
```

Should this be updated to Istanbul as the default?

**Comment by evertonfraga** [comment]
We’ll soon have Muir Glacier, maybe we can just make the change altogether

**Comment by holgerd77** [comment]
Hmm, Muir Glacier is tricky, since it doesn't activate on all supported chains, wonder if we should keep the default on `Instanbul` for simplicity and not have to create various switch/exception cases.

**Comment by ryanio** [comment]
Updated and fixed with 4.1.2 release 👍 great job everyone.

I think we should consider moving the "Fork Support" section further down from the top of the README,  it feels like it interrupts the flow of introducing the repository. Perhaps before or after section "Internal Structure".

@marcgarreau ^ a good docs task we can keep in mind.
